lib/utils/report.cjs

```diff
@@ -193,7 +203,7 @@ function isDisabled(ruleName, startLine, disabledRanges) {
 	return false;
 }
 
-/** @param {Problem & { line: number }} problem */
+/** @param {ProblemWithNormalizedPositions & { line: number }} problem */
```

Comment:
If you keep `isProblemWithNormalizedPositions`, then you can remove `& { line: number }`.

Reply:
Good catch!
Thank you